src/ContaoManager/Plugin.php

```diff
@@ -72,8 +72,8 @@ public function registerContainerConfiguration(LoaderInterface $loader, array $m
      */
     public function getRouteCollection(LoaderResolverInterface $resolver, KernelInterface $kernel)
     {
-        return $resolver
-            ->resolve(__DIR__.'/../Resources/config/routing.yml')
-            ->load(__DIR__.'/../Resources/config/routing.yml');
+        $file = __DIR__.'/../../config/routes.yaml';
+
+        return $resolver->resolve($file)->load($file);
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33